src/neptune/internal/backends/hosted_neptune_backend.py

```diff
@@ -1092,16 +1066,19 @@ def _get_file_set_download_request(self, container_id: str, container_type: Cont
             raise FetchAttributeNotFoundException(path_to_str(path))
 
     @with_api_exceptions_handler
-    def _get_column_types(self, project_id: UniqueId, column: str, types: Optional[Iterable[str]] = None) -> List[Any]:
+    def _get_column_types(self, project_id: UniqueId, column: str) -> List[Any]:
         params = {
             "projectIdentifier": project_id,
-            "search": column,
-            "type": types,
-            "params": {},
+            "query": {
+                "attributeNameFilter": {"mustMatchRegexes": [column]},
+                "experimentIdsFilter": None,
```

Comment:
Why set this explicitly when many other params are not set?

Reply:
No reason tbh - good catch, fixing